learning_wavelets/evaluation_scripts/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,50 @@
+from learning_wavelets.config import CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_bsd68
+from learning_wavelets.evaluate import METRIC_FUNCS
+from learning_wavelets.evaluate import Metrics 
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+import tensorflow as tf
+from tqdm import tqdm
```

Comment:
watch out the import order and make sure you understand clearly [the pep8 guidelines](https://www.python.org/dev/peps/pep-0008/#imports):

the third-party imports must be before the in-package imports

Reply:
and same for the other files